lib/perl/Genome/Model/Tools/Bedpe.pm

```diff
@@ -0,0 +1,30 @@
+package Genome::Model::Tools::Bedpe;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::Bedpe {
+    is => ['Command'],
```

Comment:
Aye, `Command` is `Command::V1`.  For `Command::V2`, the concept of "parent of subcommands" was split out into `Command::Tree`.  (In `V1` we could often have something be both the base class for a tree of commands and the parent of it, but now they must be distinct.)